src/components/Pokemon.js

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect, useState} from 'react';
+import './Pokemon.css';
+import axios from "axios";
+
+function Pokemon({endpoint}) {
+    const [pokemon, setPokemon] = useState({});
```

Comment:
Ik zou de default waarde hiervan `undefined` maken. Dan heb je een duidelijk verschil of de Pokémon wél of niet opgehaald is/kon worden
```suggestion
    const [pokemon, setPokemon] = useState(undefined);
```